src/common/ipc/cluster/index.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
We already have `common/cluster-ipc.ts`. How it differs from that?

Also if you want to have dedicated folder for `ipc` I would prefer to name files as you put for parent folders, instead of `index.ts` => gives better direct navigation/search from IDE.

So the final files would be:
```
common/ipc/cluster.ipc.ts
common/ipc/update-available.ipc.ts
```

Wdyt?

Reply:
Much better layout, good idea.